src/app/store/technical-records/actions/technical-record-service.actions.ts

```diff
@@ -29,6 +29,14 @@ export const getByAll = createAction(`${prefix} getByAll`, props<{ all: string }
 export const getByAllSuccess = createOutcomeAction('getByAll', true);
 export const getByAllFailure = createOutcomeAction('getByAll');
 
+export const putUpdateTechRecords = createAction(`${prefix} putUpdateTechRecords`, props<{ systemNumber: string, oldStatusCode?: StatusCodes  }>());
```

Comment:
The words `POST` and `PUT` are only used in the context of HTTP.

Suffixing these with `create` and `update` would suffice.

e.g. `updateTechRecords` & `createProvisionalTechRecord`.

It is industry standard that creation requests send a POST and updating is either a PUT or a PATCH depending on business needs.